src/pages/about/contact.js

```diff
@@ -0,0 +1,34 @@
+// Step 1: Import React
+import * as React from 'react'
+import Layout from '../../components/Layout'
+import { AboutSidebarLinks } from '../../helpers/constants'
+
+const ContactPage = () => 
+  <Layout pageTitle="Contact" sidebarLinks={AboutSidebarLinks} activeHeaderLink="About">
+    {/* <p><strong>Press Contact: <a href='environmentalenforcementwatch@gmail.com'>environmentalenforcementwatch@gmail.com</a></strong></p> */}
+    <h1>Contact</h1>
+      <form action="https://submit-form.com/y28zl2Mj">
```

Comment:
I assume this is the part that makes it work? Would be great to get a bit more comment here re how to change/update if needed